<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Exposure</class>
 <widget class="QDialog" name="Exposure">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>978_exposure</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../myqrc.qrc">
    <normaloff>:/image/978icon.png</normaloff>:/image/978icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">border-color: rgb(255, 44, 65);</string>
  </property>
  <widget class="picView" name="pvw" native="true">
   <property name="geometry">
    <rect>
     <x>75</x>
     <y>101</y>
     <width>600</width>
     <height>500</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>600</width>
     <height>500</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>500</width>
     <height>500</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>230</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>3ds Light</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>当前曝光值：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_exposure">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>230</y>
     <width>150</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>3ds Light</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_eo_add">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>190</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>3ds Light</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_eo_sub">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>270</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>3ds Light</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>3ds Light</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>图像来源：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>30</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>3ds Light</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>曝光测试算法：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_alg">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>30</y>
     <width>311</width>
     <height>32</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>3ds Light</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>ALG_TEST</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ALG_DUTTrayLocation</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ALG_SocketLocation</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ALG_DUTSocketLocation</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ALG_PocketLocation</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ALG_TrayLocation</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>380</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>3ds Light</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>测试结果：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_test_res">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>380</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>3ds Light</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>false</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_manual">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>470</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>3ds Light</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>手动测试</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_auto">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>540</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>3ds Light</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>自动测试</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_cam">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>200</width>
     <height>32</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>3ds Light</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>本地图片</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton_one">
   <property name="geometry">
    <rect>
     <x>326</x>
     <y>636</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>90</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>90</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>3ds Light</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>单张</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_realtime">
   <property name="geometry">
    <rect>
     <x>141</x>
     <y>636</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>90</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>90</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>3ds Light</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>实时</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_load">
   <property name="geometry">
    <rect>
     <x>511</x>
     <y>636</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>90</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>90</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>3ds Light</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>浏览</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_save">
   <property name="geometry">
    <rect>
     <x>817</x>
     <y>636</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>90</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>90</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>3ds Light</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>picView</class>
   <extends>QWidget</extends>
   <header location="global">picview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../myqrc.qrc"/>
 </resources>
 <connections/>
</ui>
